talks/web3/index.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+		<title>KITCTF - Smartcontract Exploitation</title>
+
+		<link rel="stylesheet" href="../reveal/dist/reset.css">
+		<link rel="stylesheet" href="../reveal/dist/reveal.css">
+		<link rel="stylesheet" href="../reveal/dist/theme/black.css">
+
+		<!-- Theme used for syntax highlighted code -->
+		<link rel="stylesheet" href="../reveal/plugin/highlight/monokai.css">
+	</head>
+	<body>
+		<div class="reveal">
+			<div class="slides">
+        <section data-separator="^(\r\n?|\n)---(\r\n?|\n)$" data-separator-vertical="^(\r\n?|\n)----(\r\n?|\n)$" data-markdown="./slides.md">
```

Comment:
Why is there a question mark, behind the "\n"?
Where is a single carriage return considered a valid line break?

Reply:
Good question. I just ripped that entire line from the codimd codebase